resources/repairnator-schema-new.json

```diff
@@ -975,6 +982,54 @@
             "Apr 5, 2018 6:12:38 AM CET"
           ]
         },
+        "machineInfo": {
```

Comment:
machineInfo should be a required property, no?

Reply:
Done.